README.html

```diff
@@ -101,6 +101,7 @@ <h2 id="changes">Changes</h2>
 <h3 id="changes-1.0.2">1.0.2</h3>
 <ul>
     <li>Prevent DEBUG txwsman logs when not at DEBUG log level. (SVC-920)</li>
+    <li>Add ability to configure Event Class for WSMANDataSourcePlugin. (ZPS-3371)</li>
```

Comment:
Please change the message to "Add ability to configure Event Class for WSMAN Data Source. (ZPS-3371)"